test/libsolidity/InlineAssembly.cpp

```diff
@@ -81,7 +81,7 @@ std::optional<Error> parseAndReturnFirstError(
 		{
 			string errors;
 			for (auto const& err: stack.errors())
-				errors += SourceReferenceFormatter::formatErrorInformation(*err);
+				errors += SourceReferenceFormatterHuman::formatErrorInformation(*err);
```

Comment:
Why do the others also use `, true, false` in this doesn't?

Reply:
This is for testing and the defaults are fine. I used the explicit settings to turn off colour where desired.

Reply:
Ok